src/main/java/org/openstreetmap/atlas/geography/atlas/raw/creation/RawAtlasGenerator.java

```diff
@@ -337,8 +349,15 @@ private Atlas rebuildAtlas(final Atlas atlas, final Set<Long> pointsToRemove,
         atlas.relations().forEach(relation ->
         {
             final RelationBean bean = new RelationBean();
-            relation.members().forEach(member -> bean.addItem(member.getEntity().getIdentifier(),
-                    member.getRole(), member.getEntity().getType()));
+            relation.members().forEach(member ->
+            {
+                final long memberIdentifier = member.getEntity().getIdentifier();
+                if (!pointsToRemove.contains(memberIdentifier))
```

Comment:
Do we need to make sure that the relation member entity is a point?

Reply:
Good point, there might be a slim chance that there is a line member with the same identifier.